### PR TITLE
Updated contributing guidelines

### LINKED ISSUE #747: [Documentation Request] Fix contribution guidelines
**Description:**

1. Contributing overview is missing the CNCF recommended section on pull request review process.
2. Contributing a new K8s stack is missing a section on updating the RBAC profile, which is needed after a new K8s stack has been added.

Fix the contributing documentation to address  both of the above.

### PR DESCRIPTION
**Pull request by sriumcp**
Signed-off-by: Srinivasan Parthasarathy <spartha@us.ibm.com>

Fixes #747

### CODE REVIEW

File: .github/workflows/mike.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Create new version
       run: |
         git config --global user.name "Srinivasan Parthasarathy"
-        git config --global user.email 5309125+sriumcp@users.noreply.github.com   
+        git config --global user.email spartha@us.ibm.com
```
Comment (kalantar) [inline .github/workflows/mike.yaml:31]: No objection but perhaps use the iter8-tools email?

Comment (sriumcp) [reply .github/workflows/mike.yaml:31]: Yes, agreed. We should use the iter8.tools email... can handle in a subsequent PR.

Comment (sriumcp) [reply .github/workflows/mike.yaml:31]: https://github.com/iter8-tools/iter8/issues/749 issue created

File: mkdocs/docs/contributing/newk8sstack.md [hunk 2]
```diff
@@ -127,5 +127,8 @@ cd iter8-<your stack>
 
     You can also refer to the [KFServing](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-kfserving), [Knative](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-knative), and [Istio](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-istio) examples.
 
-## Step 4: Submit PR
+## Step 4: Update RBAC rules
+Update the [RBAC rules that are applied to the Kubernetes cluster](../../getting-started/install/#rbac-rules) as part of the Iter8 installation.
```
Comment (kalantar) [inline mkdocs/docs/contributing/newk8sstack.md:132]: This seems to be an invalid URL. Are there needed changes to install.md?

File: mkdocs/docs/contributing/newk8sstack.md [hunk 3]
```diff
@@ -127,5 +127,8 @@ cd iter8-<your stack>
 
     You can also refer to the [KFServing](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-kfserving), [Knative](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-knative), and [Istio](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-istio) examples.
 
-## Step 4: Submit PR
+## Step 4: Update RBAC rules
```
Comment (kalantar) [inline mkdocs/docs/contributing/newk8sstack.md]: I notice we start numbering with "0". This is different from our tutorials. Perhaps start at "1"

**Review by kalantar — CHANGES_REQUESTED** [review]

**Review by huang195 — APPROVED** [review]

**Pull request merged**